src/admin/components/UserResumeTable/index.tsx

```diff
@@ -79,11 +110,28 @@ const UserResumeTable: React.FC<UserResumeTableProps> = (props) => {
     {
       title: 'Planned Graduation',
       dataIndex: 'user.graduationYear',
+
+      filterMultiple: true,
+      filters: yearFilters, // [{ text: "2016", value: "2016" }],
+
+      onFilterDropdownVisibleChange: (visible) => {
```

Comment:
Is this function necessary? If it is, can you explain why?

Reply:
onFilterDropdownVisibleChange was there to delay the getting of resumes so they would not return null

Reply:
I'm going to reopen this comment so I can take a look at it in the codebase later. There should be a cleaner approach to fetch the resumes or potentially have a null check on the resumes being defined but we can go with this if I don't come up with any other suggestions